src/components/Checkbox/Checkbox.tsx

```diff
@@ -13,6 +14,7 @@ export const Checkbox: FunctionComponent<CheckboxProps> = ({
 	id,
 	disabled = false,
 	defaultChecked = false,
+	className = '',
```

Comment:
Waarvoor is het doorgeven van die className nodig?

Reply:
dat werd gebruik in de checkbox group
nu niet meer
kzal het weg doen